.rat-excludes

```diff
@@ -93,3 +93,4 @@ INDEX
 .lintr
 gen-java.*
 .*avpr
+services/*
```

Comment:
Can this be `META-INF/services/*`? I can see someone creating a package with actual source files called `services`.